src/app/business/block/cloud-file-share/create/cloud-file-share-create.component.ts

```diff
@@ -318,6 +375,11 @@ export class CloudFileShareCreateComponent implements OnInit{
         if(value['tags']){
             dataArr['tags'] = value['tags'];
         }
+        if(value['protocols']){
+            var protoArr:string[];
+            protoArr = [value['protocols']]
```

Comment:
what is the value that protocols is expecting in the backend?
is it an array of one element or an array of strings?

Reply:
Its an Array of string. It can have multiple values also.

Reply:
In this instead of building the array we can just loop through the `value['protocols ']` and the push() method.
The current code will only add the one value always.

Reply:
Yes, Protocols can be enhanced for drop-down menu.
We can enhance and track the FS enhancement support separately.
#453